doc/docs/dev-guide/faq.md

```diff
@@ -182,3 +182,57 @@ $ git commit -s -m "This is my commit message"
 ```
 
 That’s it. Git adds your sign-off message in the commit message, and you contribution (commit) is now DCO compliant.
+
+---
+
+## How to generate Python requirements file with hashes?
+
+To generate a Python requirements.txt file with hashes, which ensures that the same versions of packages are installed across different environments, you can use the `pip-compile` tool from the `pip-tools` package. Here's a step-by-step guide on how to achieve this:
+
+### Steps:
+
+1. **Install pip-tools:**
+   First install pip-tools to manage you requirements.txt and add hashes.
```

Comment:
Place 1 empty line between title and text.

Reply:
Done